packages/react/src/components/organisms/GenTeaser/GenTeaser.md

```diff
@@ -4,36 +4,36 @@ The GenTeaser component provide a flexible and extensible content container with
 
 *Usage:*
 ```
-<GenTeaser onClick={(e) => action(e)} onKeyDown={(e) => action(e)}>
-  <GenTeaser.Details>
-    <GenTeaser.Eyebrow eyebrow={dataset.eyebrow} />
-    <GenTeaser.Button button={dataset.button} />
-    <GenTeaser.Stat>{dataCatalog.stat}</GenTeaser.Stat>
-    <GenTeaser.Title title={dataset.title} />
-    <GenTeaser.Emphasis>
-      <GenTeaser.Date date={dataset.date} />
-      <GenTeaser.Orgs orgs={dataset.orgs} />
-    </GenTeaser.Emphasis>
-    <GenTeaser.Description description={dataset.description} />
-    <GenTeaser.SearchBar search={dataCatalog.search} />
-    <GenTeaser.SubLinks>
+<GenTeaserContainer onClick={(e) => action(e)} onKeyDown={(e) => action(e)}>
```

Comment:
Can we still call the wrapper `GenTeaser`? This is to follow the naming convention. For example, see MUI https://mui.com/material-ui/api/card/ and ReachUI https://reach.tech/combobox